### PR TITLE
Add support to set user message

### PR DESCRIPTION
**Pull request by tzvatot**
@zgalor @vkareh PTAL

### CODE REVIEW

File: errors.go [hunk 1]
```diff
@@ -152,6 +152,12 @@ func GetUserMessage(err error) string {
 	return ""
 }
 
+
+// SetUserMessage sets a user readable error message.
+func (c *customError) SetUserMessage(err error, msg string) {
```
Comment (vkareh) [inline errors.go:157]: This `err` is not used. Can we remove it?

**Comment by vkareh** [comment]
Thanks for fixing typos :)

File: errors.go (hunk 1 above)
Comment (zgalor) [reply errors.go:157]: You should actually use ErrorType as a receiver.
Look at `Set` - it's a good example.

**Review by zgalor — CHANGES_REQUESTED** [review]
Also please add a test for this new func

File: errors.go (hunk 1 above)
Comment (tzvatot) [reply errors.go:157]: What is `Set` doing? the documentation doesn't really helps and I don't see any use for it in our code.

**Comment by tzvatot** [comment]
Closing this in favor of https://github.com/zgalor/weberr/pull/7

**Pull request closed without merging**